parm/config/config.ufs

```diff
@@ -0,0 +1,361 @@
+      skip_cice6=false
+      ;;
+    "--ww3")
+      ww3_res="$2"
+      skip_ww3=false
+      ;;
+    *)    # unknown option
+      echo "FATAL ERROR: Unknown option: ${key}, ABORT!"
+      exit 1
+      ;;
+  esac
+  shift
+  shift
+done
+
+case "${machine}" in
+  "WCOSS2")
+    export npe_node_max=128
+    ;;
+  "HERA" | "ORION")
+    export npe_node_max=40
+    ;;
+  "JET")
+    case "${PARTITION_BATCH}" in
+      "xjet")
+        export npe_node_max=24
+        ;;
+      "vjet" | "sjet")
+        export npe_node_max=16
+        ;;
+      "kjet")
+        export npe_node_max=40
+        ;;
+      *)
+        echo "FATAL ERROR: Unsupported PARTITION_BATCH = ${PARTITION_BATCH}, ABORT!"
+        exit 1
+        ;;
+    esac
+    ;;
+  "S4")
+    case "${PARTITION_BATCH}" in
+      "s4")
+        export npe_node_max=32
+        ;;
+      "ivy")
+        export npe_node_max=20
+        ;;
+      *)
+        echo "FATAL ERROR: Unsupported PARTITION_BATCH = ${PARTITION_BATCH}, ABORT!"
+        exit 1
+        ;;
+    esac
+    ;;
+esac
+
+# (Standard) Model resolution dependent variables
+case "${fv3_res}" in
+    "C48")
+        export DELTIM=1200
+        export layout_x=1
+        export layout_y=1
+        export layout_x_gfs=1
+        export layout_y_gfs=1
+        export nth_fv3=1
+        export nth_fv3_gfs=1
+        export cdmbgwd="0.071,2.1,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
+        export WRITE_GROUP=1
+        export WRTTASK_PER_GROUP=2
+        export WRITE_GROUP_GFS=1
+        export WRTTASK_PER_GROUP_GFS=2
+        export WRTIOBUF="1M"
+        ;;
+    "C96")
+        export DELTIM=600
+        export layout_x=2
+        export layout_y=2
+        export layout_x_gfs=2
+        export layout_y_gfs=2
+        export nth_fv3=1
+        export nth_fv3_gfs=1
+        export cdmbgwd="0.14,1.8,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
+        export WRITE_GROUP=1
+        export WRTTASK_PER_GROUP=4
+        export WRITE_GROUP_GFS=1
+        export WRTTASK_PER_GROUP_GFS=4
+        export WRTIOBUF="4M"
+        ;;
+    "C192")
+        export DELTIM=450
+        export layout_x=4
+        export layout_y=6
+        export layout_x_gfs=4
+        export layout_y_gfs=6
+        export nth_fv3=1
+        export nth_fv3_gfs=2
+        export cdmbgwd="0.23,1.5,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
+        export WRITE_GROUP=1
+        export WRTTASK_PER_GROUP=64
+        export WRITE_GROUP_GFS=2
+        export WRTTASK_PER_GROUP_GFS=64
+        export WRTIOBUF="8M"
+        ;;
+    "C384")
+        export DELTIM=${DELTIM:-200}
+        export layout_x=6
+        export layout_y=8
+        export layout_x_gfs=${layout_x_gfs:-8}
+        export layout_y_gfs=${layout_y_gfs:-12}
+        export nth_fv3=1
+        export nth_fv3_gfs=${nth_fv3_gfs:-2}
+        export cdmbgwd="1.1,0.72,1.0,1.0"  # mountain blocking, ogwd, cgwd, cgwd src scaling
+        export WRITE_GROUP=2
```

Comment:
For C384, layout (8,6) hourly output, 1 write_group 48 tasks per group should be fine.

Reply:
updated the write group and tasks per group.